### ISSUE TITLE
Training SCNN

### ISSUE THREAD

**Issue by chokkarapu** [cardwing/Codes-for-Lane-Detection#3]
Hi 
While training the model (train_lanenet.py), I could see for each and every training epoch you are doing validation epochs 1209 in number ( epoch_val in range(int(9675 / 8.0)) ) .

So what is logic behind this logic? can we do only 1-val epoch per 1-train epoch?

**Comment by cardwing** [comment]
The validation process is used for selecting hyper-parameters like learning rate etc. I will continue cleaning the codes and make them readable soon.

**Comment by chokkarapu** [comment]
> 9675 is the number of images used for validation (CULane). The whole validation process is performed every 10 K training episodes.

Thanks for information. 
But in the code there is no condition like that validation should trigger for every 10K training episodes?
Can you kindly mention in which portion of code of lines need to insert the condition you are talking about?

**Comment by chokkarapu** [comment]
I got the code logic where the validation process is triggered after 10K training episodes only
"
if epoch % 10000 != 0 or epoch == 0:
     continue
"
